recipes/cryptominisat/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -eu -o pipefail
+
+cd "$SRC_DIR"
+
+mkdir -p build && cd build
+cmake \
+  "-DCMAKE_INSTALL_PREFIX=$PREFIX" \
+  -DSTATICCOMPILE=ON \
+  -DENABLE_PYTHON_INTERFACE=OFF \
+  -DENABLE_TESTING=OFF \
```

Comment:
Does testing take a lot of time? If not, it's better to enable it and test

Reply:
It adds `lit` as a requirement, not sure if we want that.

Reply:
We'd like to have `lit` in conda-forge. https://github.com/conda-forge/staged-recipes/issues/4630

Reply:
Ok. So enabling this is not even an option at the moment…